src/asmcnc/apps/maintenance_app/widget_maintenance_spindle_settings.py

```diff
@@ -225,33 +344,84 @@ def __init__(self, **kwargs):
+            self.cooldown_speed_slider.value = self.m.yeti_cooldown_rpm_default
 
         if 'manual' in self.spindle_brand.text:
-            self.spindle_cooldown_speed.disabled = True
+            self.cooldown_speed_slider.disabled = True
 
         if 'digital' in self.spindle_brand.text:
-            self.spindle_cooldown_speed.disabled = False
+            self.cooldown_speed_slider.disabled = False
 
         self.rpm_override = False  
 
+    # def get_uptime(self):
+    #     if self.m.theateam() and self.m.get_dollar_setting(51):
+    #         if self.m.state().startswith('Idle'):
+    #             self.wait_popup = popup_info.PopupWait(self.sm, self.l)
+    #             self.m.s.write_command('M3 S0')
+    #             Clock.schedule_once(self.get_spindle_info, 0.3)
+    #         else:
+    #             popup_info.PopupError(self.sm, self.l, self.l.get_str("Please ensure machine is idle before continuing."))
+    #     else:
+    #         self.uptime_label.text = "Uptime: " + str(int(self.m.spindle_brush_use_seconds/3600)) + " hours"
+
+    # def get_spindle_info(self, dt):
+    #     self.m.s.write_protocol(self.m.p.GetDigitalSpindleInfo(), "GET DIGITAL SPINDLE INFO")
+    #     self.check_info_count = 0
+    #     Clock.schedule_once(self.check_spindle_info, 0.3)
+
+    # def check_spindle_info(self, dt):
+    #     self.check_info_count += 1
+    #     # Value of -999 represents disconnected spindle
+    #     if (self.m.s.digital_spindle_ld_qdA != -999 and self.m.s.spindle_serial_number not in [None, -999, 999]) or (self.check_info_count > 10):
+    #         self.read_restore_info()
+    #     else: # Keep trying for a few seconds
+    #         Clock.schedule_once(self.check_spindle_info, 0.3)
+
+    # def read_restore_info(self):
+    #     self.m.s.write_command('M5')
+    #     self.wait_popup.popup.dismiss()
+    #     # Value of -999 for ld_qdA represents disconnected spindle
+    #     if self.m.s.digital_spindle_ld_qdA != -999 and self.m.s.spindle_serial_number not in [None, -999, 999]:
+    #         # Get info was successful, show info
+    #         self.uptime_label.text = "Uptime: " + str(int(self.m.s.spindle_brush_run_time_seconds/3600)) + " hours"
+    #     else:
+    #         # Otherwise, spindle is probably disconnected
+    #         error_message = self.l.get_str("No SC2 Spindle motor detected.") + " " + self.l.get_str("Please check your connections.")
+    #         popup_info.PopupError(self.sm, self.l, error_message)
+
     def update_strings(self):
         self.rpm_label.text = self.l.get_str("RPM")
         self.seconds_label.text = self.l.get_str("seconds")
+        self.cooldown_settings_label.text = self.l.get_str("SPINDLE COOLDOWN SETTINGS")
+        # self.uptime_label.text = self.l.get_str("Turn on spindle to read")
+        self.min_speed_label.text = "10k " + self.l.get_str("RPM")
+        self.max_speed_label.text = "20k " + self.l.get_str("RPM")
```

Comment:
I know the wireframe said 10k / 20k, but looking at the screen I think there's plenty of space to make these 10,000 and 20,000 RPM (which will probably translate better.

Reply:
Updated:
![image](https://github.com/YetiTool/easycut-smartbench/assets/75572055/5302a7e6-87fd-41e0-b08b-611588c497e6)